### PR TITLE
qt5 Stylesheet Fixes

### PR DESCRIPTION
**Pull request by RubioJr9**
Fixed:
-black text on gray backgrounds
-button styling
-ShowFieldGlyphs spacing
-ViewScene look axis dropdown spacing

Related Tickets:
#1983

Reviewers:
@jessdtate @dcwhite

### CODE REVIEW

**Comment by RubioJr9** [comment]
Just a heads up, I redid the ShowFieldGlpyhs ui and used the proper layout boxes. I think it will do better with smushing on different configurations

**Comment by RubioJr9** [comment]
Made some changes to showfieldglyphs for mac os. Everything seems good except for coloring of groupbox contents. I wasn't able to figure that out before having to leave. I'm changed background colors in the stylesheet to set everything to gray. The groupbox style is being set as well but I couldn't figure out a way to exclude that or set it back to default. It looks right in qt designer but it set differently in the application.

**Comment by dcwhite** [comment]
Cool, thanks

**Comment by dcwhite** [comment]
Cool, thanks

File: src/Interface/Application/SCIRunMainWindow.cc [hunk 1]
```diff
@@ -102,33 +102,6 @@ SCIRunMainWindow::SCIRunMainWindow()
...
 
-  setStyleSheet(
-    "background-color: rgb(66,66,69);"
-    "color: white;"
-    "selection-color: yellow;"
-    "selection-background-color: blue;"//336699 lighter blue
-    "QToolBar {        background-color: rgb(66,66,69); border: 1px solid black; color: black;     }"
-    "QProgressBar {        background-color: rgb(66,66,69); border: 0px solid black; color: black  ;   }"
-    "QDockWidget {background: rgb(66,66,69); background-color: rgb(66,66,69); }"
-    "QPushButton {"
```
Comment (dcwhite) [inline src/Interface/Application/SCIRunMainWindow.cc]: @RubioJr9 These QPushButton stylesheet options seem to have been lost. Did they not apply anywhere so you removed them? Or were they mistakenly not copied to the app level?

Comment (RubioJr9) [reply src/Interface/Application/SCIRunMainWindow.cc]: I'll double check but I believe they didn't have any visible effect.

**Comment by RubioJr9** [comment]
Screen Shot 2019-11-07 at 2 43 10 PM
(Left is last release build; Right is this branch)

Some slight differences but the buttons don't seem to be affected

**Comment by dcwhite** [comment]
Ok cool. Let's merge it

**Comment by RubioJr9** [comment]
Turns out I took those out because they looked bad on linux, but had no effect on macs

**Comment by dcwhite** [comment]
Ah ok. We can look tomorrow

**Comment by dcwhite** [comment]
#648

**Pull request merged**